tracer/src/Datadog.Trace/Activity/OtlpHelpers.cs

```diff
@@ -507,5 +613,52 @@ internal static void SerializeEventsToJson(Span span, IEnumerable events)
 
             return null;
         }
+
+        private static bool IsAllowedAtributeType(object value)
```

Comment:
This is just a duplicate from `ActivityEventConverter` right?

After this PR we should refactor them both out into a common location to reduce the duplication

Reply:
Indeed.